platform/ui/tailwind.config.js

```diff
@@ -262,14 +263,15 @@ module.exports = {
       full: '100%',
       screen: '100vh',
     }),
-    inset: {
+    inset: theme => ({
+      ...theme('spacing'),
```

Comment:
need this for more flexibility when positioning things absolutely (e.g. bottom-2).